be/src/service/internal_service.cpp

```diff
@@ -423,200 +480,233 @@ void PInternalServiceImpl::tablet_fetch_data(google::protobuf::RpcController* co
                                              const PTabletKeyLookupRequest* request,
                                              PTabletKeyLookupResponse* response,
                                              google::protobuf::Closure* done) {
-    [[maybe_unused]] brpc::Controller* cntl = static_cast<brpc::Controller*>(controller);
-    brpc::ClosureGuard guard(done);
-    Status st = _tablet_fetch_data(request, response);
-    st.to_protobuf(response->mutable_status());
+    DorisMetrics::instance()->tablet_fetch_data->increment(1);
+    _heavy_work_pool.offer([this, controller, request, response, done]() {
```

Comment:
please Do not put this RPC interface into thread pool, it's very light weight and can't be mixed with other rpc interface, since it's latency sensitive

Reply:
before this pr ,this rpc interface also  mixed with each other 
i don't think put into thread pool will effect latency

Reply:
if the thread pool in hight CPU work load, this interface will be queued up and the latency is infected

Reply:
Other heavy tasks could block all pthread and no worker thread for this light weight rpc and infect latency

Reply:
Even if I don't add this pr, will have the same problem